src/core/buffer.c

```diff
@@ -320,6 +320,143 @@ JANET_CORE_FN(cfun_buffer_chars,
     return argv[0];
 }
 
+static int should_reverse_bytes(const Janet *argv, int32_t argc) {
+    JanetKeyword order_kw = janet_getkeyword(argv, argc);
+    if (!janet_cstrcmp(order_kw, "le")) {
+#if JANET_BIG_ENDIAN
+        return 1;
+#endif
+    } else if (!janet_cstrcmp(order_kw, "be")) {
+#if JANET_LITTLE_ENDIAN
+        return 1;
+#endif
+    } else if (!janet_cstrcmp(order_kw, "native")) {
```

Comment:
`int/to-bytes` used `nil` to represent native endianness. I felt `:native` was more descriptive but I would understand the desire to maintain consistency with what already exists.

Reply:
Another path to consistency might be for `int/to-bytes` to also support `:native`?